lectures/10.HPC_and_parallel_compute/SLURM_cheatsheet.md

```diff
@@ -0,0 +1,79 @@
+# Slurm Guide
```

Comment:
Should there be a line here suggesting how to implement SBATCH commands? I imagine that you submit a file that includes these commands? I believe you imply this in the `Submitting a job` section, but it would be good to spell it out explicitly.

Reply:
added some explanation to this files